### DISCUSSION TITLE
Set user profile picture

### DISCUSSION THREAD

**Discussion by gg-ipp** [dagster-io/dagster#19134] [Q&A]
Hi,

How can I set my user profile picture on Dagster cloud UI?

It seems possible, some of my colleagues have pictures, but I can't find the option. Tried 'User settings', 'Organization settings' -> 'Users', and setting a Gravatar.

The version we use: can't find it on the UI, the project file says 1.5.13.

Best,
George

**Comment by jmsanders** [comment]
The answer depends on your login method. If you're logging in via any of our supported SSO methods (Google, GitHub), we'll automatically sync your latest profile picture from the source. If you're logging in via email or SCIM, updating profile pictures isn't supported yet.
